Docker/test/selenium/package.json

```diff
@@ -19,7 +19,7 @@
 		"npm-run-all": "^4.1.5",
 		"stylelint-config-wikimedia": "0.10.1",
 		"wdio-mediawiki": "^1.0.0",
-		"wdio-wikibase": "^4.0.1",
+		"wdio-wikibase": "^5.1.0",
 		"webdriverio": "^6.1.16",
```

Comment:
I think this line needs to go, it seems to not be picking the correct webdriverio version

Reply:
https://github.com/wmde/wikibase-release-pipeline/pull/211/checks?check_run_id=3484803294#step:4:284